src/Common/AsynchronousMetrics.cpp

```diff
@@ -1135,7 +1137,7 @@ void AsynchronousMetrics::update(TimePoint update_time, bool force_update)
 
                 Float64 num_cpus_to_normalize = max_cpu_cgroups > 0 ? max_cpu_cgroups : num_cpus;
 
-                if (num_cpus_to_normalize > 0)
+                if (num_cpus_to_normalize > 0 && !cgroup_cpu_metrics_present)
```

Comment:
why? :)

Reply:
have you saw cpu normalized not reported on a cgroups v1 host?

Reply:
I didn't check if it was v1 or v2. But they are not there for sure: https://github.com/ClickHouse/ClickHouse/issues/76810  (there is even a simple test showing it it fiddle).